packages/sdks/snippets/gen1-react/src/routes/targeted-page/custom-targeting-request.tsx

```diff
@@ -0,0 +1,11 @@
+import { builder } from '@builder.io/react';
+import type { GetContentOptions } from '@builder.io/sdk';
+
+export default function targetedRequest({ model, url }: GetContentOptions) {
+  return builder.get(model!, {
+    userAttributes: {
+      audience: ['mens-fashion'],
+      urlPath: url,
+    },
+  });
+}
```

Comment:
- Recently added `!` to the list of things we want to avoid in our docs: https://www.notion.so/builderio/Docs-code-snippet-guidelines-1283d7274be580d1a96cdafcf6cab708?pvs=4#1ab3d7274be58085b038e2b7a609c435

- `GetContentOptions` is specific to the second argument of `builder.get`, so it's more consistent to use it only for that part
- `url` in gen1 can be used to replace `userAttributes.urlPath`, as per its code comment: 
![CleanShot 2025-03-04 at 14 53 43@2x](https://github.com/user-attachments/assets/ff47b8b4-d6f8-45cd-a054-1112b401edf6)

Putting all that together:
```suggestion
export default function targetedRequest(modelName: string, options: GetContentOptions) {
  return builder.get(modelName, {
    userAttributes: {
      audience: ['mens-fashion'],
    },
    ...options,
  });
}
```